_data/chains/62.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Ethereum Classic Testnet",
+  "name": "Ethereum Classic Testnet Ropsten",
```

Comment:
What? That is Morden...

Reply:
Ha ha - brain fart.   I will make a new PR.